TA_Scheduler/Tests/LoginTests/test_loginAcceptance.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase, Client
```

Comment:
Would recommend merging rather fast-forwarding in git to prevent all these extra files from showing up in pull requests

Reply:
What do you mean by fast-forwarding?  I was basically only changing my three files, and merging main onto my branch when changes that affected me were made.

Reply:
earlier, your git history looked like this 
![image](https://user-images.githubusercontent.com/45466247/144921825-124ebd4f-31f1-4009-94b1-1dd0e405f2c4.png)
which is not the result of merging. which looks like this 
![image](https://user-images.githubusercontent.com/45466247/144921916-66ba52a5-6c10-4f72-b239-179d79167ee1.png)

I think that's where a bunch of issues that we don't use anymore is coming from. Git thinks you made these new changes, but stuff like the home page we delete a long time ago.